bazel/repositories.bzl

```diff
@@ -399,6 +400,25 @@ def _com_google_absl():
         name = "abseil_optional",
         actual = "@com_google_absl//absl/types:optional",
     )
+    native.bind(
+        name = "abseil_synchronization",
+        actual = "@com_google_absl//absl/synchronization:synchronization",
+    )
+    native.bind(
+        name = "abseil_time",
+        actual = "@com_google_absl//absl/time:time",
+    )
+
+def _com_googlesourcecode_cctz():
+    _repository_impl("com_googlesource_code_cctz")
+    native.bind(
+        name = "abseil_time_zone",
```

Comment:
How come there are two binds here for the same target name? This and below.

Reply:
will attempt to clean up.